ui/components/logs/empty-state.tsx

```
@@ -215,16 +214,15 @@ const response = await completion({
 		<div className="flex w-full flex-col items-center justify-center space-y-8">
-			<div className="space-y-2 text-center">
-				<h2 className="text-3xl font-bold">Welcome to Request Logs</h2>
-				<p className="text-muted-foreground text-lg">Monitor and analyze all your API requests in real-time</p>
-			</div>
-
 			{isSocketConnected && (
-				<div className="flex items-center justify-center gap-2 text-sm">
-					<RefreshCw className="h-4 w-4 animate-spin" />
+				<div className="mx-auto inline-flex items-center bg-green-50 border border-green-200 rounded-full px-3 sm:px-4 py-2 text-green-700 text-xs sm:text-sm font-medium mb-4 sm:mb-6">
+				<span className="relative flex h-2 w-2 mr-2 sm:mr-3">
+				  <span className="animate-ping absolute inline-flex h-full w-full rounded-full bg-green-500 opacity-75"></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)